Sources/Plasma/PubUtilLib/plPipeline/plTextFont.cpp

```diff
@@ -57,13 +57,17 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "plDebugText.h"
 
+#include <ft2build.h>
+#include FT_FREETYPE_H
+#include FT_GLYPH_H
+
+#include <memory>
+
 #if defined(HS_BUILD_FOR_APPLE)
 #import <CoreGraphics/CoreGraphics.h>
 #import <CoreText/CoreText.h>
 #endif
 
-#define DisplayableChar(c) (c >= 0 && c <= 128)
```

Comment:
This check was only being run on the char type so the compiler advised me it was redundant. Kind of wondering if that's true? This only goes to 128.